python/ray/data/tests/test_dataset.py

```diff
@@ -59,6 +59,7 @@ def _read_stream(self, f: "pa.NativeFile", path: str, **reader_args):
 # https://github.com/ray-project/ray/issues/20625
 @pytest.mark.parametrize("block_split", [False, True])
 def test_bulk_lazy_eval_split_mode(shutdown_only, block_split, tmp_path):
+    ray.shutdown()
```

Comment:
wondering why we need to add it here?

Reply:
It may be up to the test environment and/or other tests, which left a cluster.